modules/apps/transfer/keeper/msg_server.go

```diff
@@ -71,22 +72,14 @@ func (k Keeper) UpdateParams(goCtx context.Context, msg *types.MsgUpdateParams)
 // the unwound path to take. It assumes that only a single token is present (as this is verified in ValidateBasic)
 // in the tokens list and ensures that the token is not native to the chain.
 func (k Keeper) unwindHops(ctx sdk.Context, msg *types.MsgTransfer) (*types.MsgTransfer, error) {
-	coins := msg.GetCoins()
-	token, err := k.tokenFromCoin(ctx, coins[0])
+	fullTokenHops, err := k.getUnwindHops(ctx, msg.GetCoins())
```

Comment:
```suggestion
	unwindHops, err := k.getUnwindHops(ctx, msg.GetCoins())
```
I actually found the previous naming to be more readable?